app/kumactl/cmd/apply/apply.go

```diff
@@ -201,8 +201,10 @@ func parseResource(bytes []byte) (model.Resource, error) {
 var _ model.ResourceMeta = &meta{}
 
 type meta struct {
-	Name string
-	Mesh string
+	Name             string
+	Mesh             string
+	CreationTime     time.Time
+	ModificationTime time.Time
```

Comment:
Is it needed though? I think we can revert this file to master

Reply:
but not sure if I revert it to master, `GetCreationTime()` is `time.Unix` and it will be different for each machine and the test will fail in ci.

Reply:
`time.Unix(0, 0)` indicates the constant time

Reply:
changed